### PR TITLE
Updated "null values" in ImportClinicalData.java and validateData.py

### PR DESCRIPTION
**Pull request by angelicaochoa**
# What? Why?

Fix # .

Changes proposed in this pull request:
- a
- b
# Checks
- [ ] Runs on Heroku.
- [ ] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.
# Any screenshots or GIFs?

If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).
# Notify reviewers

If your pull request involves changes to an existing file, one or more of the
people that edited before you might be good candidates to review it. Please use
`git blame <filename>` to determine that and notify them here. Otherwise notify
everybody in the core team:
@cBioPortal/core-developers

### CODE REVIEW

**Comment by angelicaochoa** [comment]
@pieterlukasse

File: core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java [hunk 1]
```diff
@@ -70,7 +70,10 @@
         DISCREPANCY("Discrepancy"),
         COMPLETED("Completed"),
         NULL("null"),
-        MISSING("");
+        MISSING(""),
+        NA("NA"),
+        N_A("N/A"),
+        UNKNOWN("unknown");
```
Comment (pieterlukasse) [inline core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java]: would be good to add a unit test for this. See also https://github.com/cBioPortal/cbioportal/pull/1398/commits/392edfc97fafccf957ceb3cb5ac1682e2129c360

You will discover that your change for `N/A` does *not* work and that `N/A` will only work if you also make changes to the `MissingAttributeValues.has` function below.

File: core/src/main/scripts/importer/validateData.py [hunk 2]
```diff
@@ -1195,7 +1195,7 @@ class ClinicalValidator(Validator):
 
     REQUIRE_COLUMN_ORDER = False
     PROP_IS_PATIENT_ATTRIBUTE = None
-    NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", ""]
+    NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", "", "[unknown]", "unknown", "NA", "N/A"]
     ALLOW_BLANKS = True
```
Comment (pieterlukasse) [inline core/src/main/scripts/importer/validateData.py]: please add these scenarios to the corresponding pyunit test. See also https://github.com/cBioPortal/cbioportal/pull/1398/commits/ca49289fc1bedd78ffb6552fb44eb1e5f3e79136 for an example.

**Comment by pieterlukasse** [comment]
hi @angelicaochoa thanks for this PR and for notifying me. General comments: the changes in python look good, but a test is missing. The change in Java needs an extra fix. See also my inline comments.

File: core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java [hunk 3]
```diff
@@ -41,6 +41,8 @@
 import java.util.*;
 import java.util.regex.*;
 import org.apache.commons.collections.map.MultiKeyMap;
+import static org.junit.Assert.assertTrue;
```
Comment (pieterlukasse) [inline core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java]: what are the junit classes doing here? Looks like a mistake

File: core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java [hunk 4]
```diff
@@ -45,6 +45,8 @@
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.test.context.transaction.TransactionConfiguration;
 import org.springframework.transaction.annotation.Transactional;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
 
 import static org.junit.Assert.*;
```
Comment (pieterlukasse) [inline core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java]: double imports?

File: core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java [hunk 5]
```diff
@@ -242,4 +244,16 @@ public void testImportClinicalDataParameters() throws Exception {
...
+     * Test the MissingAttributeValues enum.
+     * 
+     */
+    @Test
+    public void testHasMethod() {
+        assertTrue(ImportClinicalData.MissingAttributeValues.has("NA"));
+        assertTrue(ImportClinicalData.MissingAttributeValues.has("N/A"));
+        assertTrue(ImportClinicalData.MissingAttributeValues.has("unknown"));
+        assertFalse(ImportClinicalData.MissingAttributeValues.has("undefined"));
+    }
```
Comment (pieterlukasse) [inline core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java]: nice :+1:

**Comment by pieterlukasse** [comment]
please check the extra comments I added.

File: core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java (hunk 3 above)
Comment (angelicaochoa) [inline core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java]: @pieterlukasse thanks for catching that

File: core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java (hunk 4 above)
Comment (angelicaochoa) [inline core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportClinicalData.java]: @pieterlukasse and for catching this too

**Pull request closed without merging**